code/data_structures/src/queue/queue_using_stack/queue_using_stack.c

```diff
@@ -0,0 +1,108 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct stack 
```

Comment:
Consider using a `typedef` so it's easier to instantiate this.

Reply:
`typedef struct 
{
    int top;
    unsigned capacity;
    int* A;
} stack;`

Now I am getting warnings from the compiler (incompatible pointer type). Running fine.

Reply:
I think the style I used earlier for struct would be fine.